lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -3910,18 +3910,36 @@ void request_disassemble(const llvm::json::Object &request) {
     return;
   }
 
-  addr_ptr += GetSigned(arguments, "instructionOffset", 0);
```

Comment:
We still need to use the "instructionOffset" here if it is specified. Seems like it isn't optional, so if it isn't here, then maybe return an error if it is required. We might want to add a overload of GetSigned like:
```
llvm::Expected<int64_t> GetSigned(arguments, "instructionOffset");
```
This would return an error if it isn't in the args, or if the decoding failed. We can just return a string version of the error as an error to this command.